### PR TITLE
Changing some "will happen"s to "hopefully will happen"s

### PR DESCRIPTION
**Pull request by jgillula**
Made some changes to the documentation to avoid promising things that we're gonna try to do, but can't absolutely guarantee, for purposes of legal review.

### CODE REVIEW

**Comment by SwartzCr** [comment]
We actually have Jessie backports already
So it should say something like
Packages exist for Debian Jessie via backports. First you'll have to follow the instructions here[backports.debian.org/instructions] to enable the Jessie backports repo, if you have not already done so. Then run:
`apt-get install certbot python-certbot-apache -t jessie-backports`

File: README.rst [hunk 1]
```diff
@@ -128,8 +128,8 @@ System Requirements
 ===================
 
 The Let's Encrypt Client presently only runs on Unix-ish OSes that include
-Python 2.6 or 2.7; Python 3.x support will be added after the Public Beta
-launch. The client requires root access in order to write to
+Python 2.6 or 2.7; Python 3.x support will hopefully be added after the Public
+Beta launch. The client requires root access in order to write to
```
Comment (schoen) [inline README.rst]: It's a while after the Public Beta launch already, so how about making this say "hopefully be added in the future"?

**Comment by SwartzCr** [comment]
lgtm

**Comment by schoen** [comment]
I agree.

**Pull request merged**